docs/api/en/cameras/Camera.html

```diff
@@ -72,6 +72,11 @@ <h3>[method:Camera copy]( [param:Camera source], [param:Boolean recursive] )</h3
 		Copy the properties from the source camera into this one.
 		</p>
 
+		<h3>[method:null dispose]()</h3>
+		<p>
+		Clears camera related data internally cached by [page:WebGLRenderer].
```

Comment:
Clears camera-related data cached by [page:WebGLRenderer].
    You should call this method when you ___________.

Reply:
"You should call this method when you don't need the camera for rendering anymore".

How about this? We can use a similar wording for `Scene.dispose()`?

Reply:
maybe this:  

    Clears camera-related data structures cached by [page:WebGLRenderer].
    You should call this method if you don't need the camera for rendering anymore.